module/VuFind/src/VuFind/Db/Entity/UserList.php

```diff
@@ -38,55 +38,76 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:plugins:database_gateways Wiki
  *
- * @ORM\Table(name="user_list", indexes={@ORM\Index(name="user_id", columns={"user_id"})})
+ * @ORM\Table(name="user_list",
+ *          indexes={@ORM\Index(name="user_id", columns={"user_id"})}
+ * )
  * @ORM\Entity
  */
 class UserList implements EntityInterface
 {
     /**
+     * Unique ID.
+     *
      * @var int
      *
-     * @ORM\Column(name="id", type="integer", nullable=false)
      * @ORM\Id
+     * @ORM\Column(type="integer",
+     *          nullable=false
+     * )
      * @ORM\GeneratedValue(strategy="IDENTITY")
      */
     protected $id;
 
     /**
+     * Title of the list.
+     *
      * @var string
      *
      * @ORM\Column(name="title", type="string", length=200, nullable=false)
      */
     protected $title;
 
     /**
+     * Description of the list.
+     *
      * @var string|null
      *
      * @ORM\Column(name="description", type="text", length=65535, nullable=true)
      */
     protected $description;
 
     /**
+     * Creation date.
+     *
      * @var \DateTime
      *
-     * @ORM\Column(name="created", type="datetime", nullable=false, options={"default"="2000-01-01 00:00:00"})
+     * @ORM\Column(name="created",
+     *          type="datetime",
+     *          nullable=false,
+     *          options={"default"="2000-01-01 00:00:00"}
+     * )
      */
     protected $created = '2000-01-01 00:00:00';
 
     /**
+     * Flag to indicate whether or not the list is public.
+     *
```

Comment:
Might want to add `TODO: update to boolean after merge of #2689` so we don't lose track of finishing that cleanup after the PR is completed.